From 87aef06748167df25afd62498ef05427c18769e4 Mon Sep 17 00:00:00 2001
From: Gatien Chevallier <gatien.chevallier@foss.st.com>
Date: Wed, 23 Oct 2024 15:08:16 +0200
Subject: [PATCH 1294/1695] core: lib: scmi-server: update displayed SCP
 Firmware version

SCP Firmware version is 2.13.0, not 2.11.0. Change the minor version that
is displayed at boot.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git d9c4df97fc9cf1e5319d8511adbf1eb6c95da38e core: lib: scmi-server: update displayed SCP Firmware version]
Change-Id: I21abae82a56eb7339d3382186777537fca19e75e
Signed-off-by: Gatien Chevallier <gatien.chevallier@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/413581
Tested-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Domain-Review: Yann GAUTIER <yann.gautier@foss.st.com>
---
 core/lib/scmi-server/sub.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/lib/scmi-server/sub.mk b/core/lib/scmi-server/sub.mk
index 4d4637b3c..eb03d1259 100644
--- a/core/lib/scmi-server/sub.mk
+++ b/core/lib/scmi-server/sub.mk
@@ -21,7 +21,7 @@ scpfw-out-path := $(out-dir)/$(libdir)
 # This script was validated against SCP-firmware 2.11.0 development branch,
 # from commit f1d894921d76 ("product/optee-fvp: Add new OPTEE FVP product").
 scpfw-integ-version-maj = 2
-scpfw-integ-version-min = 11
+scpfw-integ-version-min = 13
 scpfw-integ-version-pat = 0
 scpfw-integ-version = $(scpfw-integ-version-maj).$(scpfw-integ-version-min).$(scpfw-integ-version-pat)
 
-- 
2.47.2

